### PR TITLE
Improve the scrolling on task lists

### PR DESCRIPTION
**Pull request by JoshSmith**
# What's in this PR?

Improves the scrolling of task lists.

### CODE REVIEW

**Comment by JoshSmith** [comment]
@snewcomer idk how this might conflict with your work on #1491

File: app/styles/components/task-list-cards.scss [hunk 1]
```diff
@@ -1,35 +1,57 @@
...
 }
 
 .task-list-cards {
-  height: 100%;
+  margin: 0 3px;
+  min-height: 10px;
+  overflow-x: hidden;
+  overflow-y: auto;
+  padding: 0 3px;
+  white-space: normal;
```
Comment (snewcomer) [inline app/styles/components/task-list-cards.scss]: I wonder if this `white-space` declarations are needed?

File: app/styles/components/task-list-cards.scss [hunk 2]
```diff
@@ -1,35 +1,57 @@
...
   background-color: #F7F7F7;
   border-radius: 4px;
-  margin-left: 7px;
+  display: flex;
+  flex-direction: column;
   max-height: 100%;
-  overflow-x: hidden;
-  overflow-y: scroll;
-  padding: 8px 6px;
+  position: relative;
```
Comment (snewcomer) [inline app/styles/components/task-list-cards.scss]: is this `position: relative` needed with no other position declared on the `.task-list`?

File: app/styles/components/task-list-cards.scss [hunk 3]
```diff
@@ -1,35 +1,57 @@
...
-    margin-bottom: 10px;
+    padding: 8px 6px;
   }
 }
 
 .task-list-cards {
-  height: 100%;
+  margin: 0 3px;
+  min-height: 10px;
+  overflow-x: hidden;
```
Comment (snewcomer) [inline app/styles/components/task-list-cards.scss]: can it `overflow-x` or is this just there for safety?

Comment (JoshSmith) [reply app/styles/components/task-list-cards.scss]: It can, yes.

File: app/styles/components/task-list-cards.scss (hunk 1 above)
Comment (JoshSmith) [reply app/styles/components/task-list-cards.scss]: Needed due to parent/child `white-space` declarations.

File: app/styles/components/task-list-cards.scss (hunk 2 above)
Comment (JoshSmith) [reply app/styles/components/task-list-cards.scss]: I believe so, thanks to some of the flex behavior on children.

**Pull request merged**